bin/host/src/lib.rs

```diff
@@ -19,3 +19,8 @@ pub use preimage::{
 
 mod server;
 pub use server::PreimageServer;
+
+pub mod cli;
+pub mod eth;
+pub mod interop;
+pub mod single;
```

Comment:
If we want to restrict the accessible types from these crates, happy to apply the `pub use` pattern above.